lib/tenstorrent/bh_arc/noc2axi.c

```diff
@@ -63,6 +61,15 @@ typedef union {
 #define RING0_TLB_REG_OFFSET     0x1000
 #define AXI2NOC_RING_SEL_BIT     15
 
+#ifdef CONFIG_EMUL
```

Comment:
Usually this implies that there is an emulation driver being used.

> [NOTE]
> There is a [dma_emul driver](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/dma/Kconfig.emul) upstream. If `noc_dma.c` simply implemented [the DMA API](https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/drivers/dma.h) (i.e. was a Zephyr driver), then we wouldn't need to specifically have a fake for NOC.

Reply:
Which Kconfig would you rather I use?

Reply:
Other bringup code uses `#if CONFIG_BOARD_NATIVE_SIM`, which is still a bit dirty.

I think the reason that `CONFIG_EMUL` was used in the test was because the `i2c_emul` driver is used.

Reply:
Not saying you should implement the DMA API in this PR, which is just a workaround for a regression, but it would have been good if the code was added that way in the first place.

Reply:
Switched to use `CONFIG_BOARD_NATIVE_SIM`